pkg/ctl/delete/cluster.go

```diff
@@ -89,13 +99,13 @@ func doDeleteCluster(cfg *api.ClusterConfig, name string) error {
 
 	if clusterErr {
 		if handleIfError(ctl.DeprecatedDeleteControlPlane(), "control plane") {
-			handleIfError(stackManager.DeprecatedDeleteStackControlPlane(waitDelete), "stack control plane")
+			handleIfError(stackManager.DeprecatedDeleteStackControlPlane(waitDelete), "stack control plane (deprecated)")
```

Comment:
At what point can we deleted the deprecated stuff? v0.2.0?

Reply:
Yes, that seems like the best plan!